Bug 1612390 sidebars included in extension.get views when window id specified

rebloor opened the pull request:
Addresses [feedback](https://github.com/mdn/content/pull/8399#discussion_r700563851) on [Sidebars included in extension.getViews when windowId specified #8399 ](https://github.com/mdn/content/pull/8399) made after the merge.

rebloor committed:
Sidebars included in extension.getViews when windowId specified

rebloor committed:
origin/Bug-1612390-sidebars-included-in-extension.getViews-when-windowId-specified

Adjusted wording based on feedback

rebloor committed:
origin/Bug-1612390-sidebars-included-in-extension.getViews-when-windowId-specified

Adjusted wording based on feedback

Rob--W reviewed (approved):
The change looks good. This PR also includes the already-merged commit. Could you rebase to `origin/main` and choose a less cryptic commit title?

StephNathai committed:
fix: replace broken packaging_and_installation link (#8567)

Fixes #8162

The packaging_and_installation link is broken on the following pages:
- https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization
- https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/debugging_(before_firefox_50)

jxu committed:
Fix CSS grouping selector description grammar (#8576)

teoli2003 committed:
Add missing "closed" state to interface overview (#8556)

pocketdr committed:
Update link of issue in MDN/Contribute/Feedback (#8552)

* Update link of issue in MDN/Contribute/Feedback

Update section at original article before translating.

* Update index.html

hamishwillee committed:
Markdown Learn/Server-Side: Table - strip class (standard-table and learn-table) (#8583)

* Table - strip class (standard-table and learn-table)

* Remove crap injected during testing

hamishwillee committed:
[markdown] Learn section - remove image styles and classes of learn box tables (#8586)

* Remove strong tag from pre block in server-side

* Remove all styling from images

* Remove the table classes learn-box standard-table from across learn

hamishwillee committed:
htmlmediaelement : default playback rate - exception if value unsupported (#8579)

* htmlmediaelement : default playback rate - throw exception if value not supported

* Removes the old note

renomureza committed:
Array flatMap: Inconsistent Parameter Name (#8584)

* Array flatMap: Inconsistent Parameter Name

In the syntax it says callbackFn, but in the parameter description it says callback: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap

* Update index.md

hamishwillee committed:
Page template to markdown: SVG and HTML elements (#8587)

* SVG page element template: move file .html to .md

* SVG Element Page template: Convert content to markdown

* HTML Element page template: move file .html to .md

* HTML Element page template: Convert content to markdown

* Fix typo

* Mark up code block to svg

wbamberg committed:
Delete TextRange (#8589)

Commit:
Adds declaration to example (#8592)

Adds pickerOpts as used here https://developer.mozilla.org/en-US/docs/Web/API/File_System_Access_API
Fixes #6984

inokawa committed:
Fix typos in Number (#8594)

teoli2003 committed:
Remove link to enum type (#8599)

Elchi3 committed:
XRSession: Replace Event table with a list (#8604)

dependabot[bot] committed:
chore(deps): bump @mdn/yari from 0.4.755 to 0.4.756 (#8609)

rebloor committed:
Bug 1669566 add cookie store id to browser.downloads (#8106)

* Bug-1669566-Add-cookieStoreId-to-browser.downloads

Initial changes adding cookieStoreId to browser.download.download, downloads.DownloadQuery (browser.download.search), and downloads.DownloadItem (browser.download.erase).

* Added cross-reference from "contextual identities" to related how to article

* Release notes update

* Updates to address feedback

mfuji09 committed:
"Array Iterator" never appear in codes (#8580)

* `Array Iterator` never appear in code

* Update files/en-us/web/javascript/reference/global_objects/array/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/web/javascript/reference/global_objects/array/values/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/web/javascript/reference/global_objects/bigint64array/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* fix markup in Web API references

The Web API hasn't been converted in Markdown...

* fix more representations of "array iterator"

I am noticed that there are still "Array Iterator" which should be fixed.
I have fixed them.

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Elchi3 committed:
WebXR landing page: Remove 404 guide links, create overview (#8607)

mikesellitto committed:
Fix typos - timout > timeout (#8621)

teoli2003 committed:
MP4 audio -> MP4 video (#8620)

Fixes #8615

digi-booster committed:
Update Outdated Url in <a>: The Anchor element (#8625)

https://a11yproject.com/posts/large-touch-targets/(old) -> https://www.a11yproject.com/posts/2018-11-21-large-touch-targets/(new)

digi-booster committed:
Removing Broken Link in XML introduction (#8626)

maybe http://www.alistapart.com/d/usingxml/xml_uses_a.html URL changed or removed.

teoli2003 committed:
Fix redirects in macros for WebGL (#8598)

Elchi3 committed:
WebXR: Update syntax blocks for methods/ctors (#8593)

Commit:
Fixes #8540 - Adds Exceptions from RegExp() (#8555)

Commit:
Fixes 7968 - mocha -> npx mocha (#8560)

AnilSeervi committed:
Update devtools links of chrome and edge (#8563)

juliandescottes committed:
WebDriver conformance changes for Firefox 92 (#8564)

digi-booster committed:
Replacing Url in scroll-snap-points-y (#8627)

* Replacing Url in scroll-snap-points-y

https://web.dev/css-scroll-snap/p (404) -> https://developers.google.com/web/updates/2018/07/css-scroll-snap (200)

* Fix url

Co-authored-by: Rachel Andrew <me@rachelandrew.co.uk>

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>
Co-authored-by: Rachel Andrew <me@rachelandrew.co.uk>

MendyBerger committed:
Added WebAssembly control flow operators (#8581)

Commit:
Fixes #6991 - removes question and associated code (#8591)

Verified that even without it the readyState is set to 2 => DONE i.e. The
operation is complete.

teoli2003 committed:
Fix typo in macro name + redirected link (#8597)

Commit:
Fixes #6593 - Updates one link and removes another (#8600)

teoli2003 committed:
Remove link to enum-like type (#8601)

teoli2003 committed:
Remove mention of dictionary's property (#8628)

mathewhodson committed:
Use https for links in XML introduction (#8630)

Also move a description closer to the link it's for.

aynugek committed:
Refine database-support language in Django “Setting up” doc (#8632)

digi-booster committed:
Update link URL for “Number” type in isNaN() doc (#8640)

digi-booster committed:
Replacing SSL Url to no SSL url in Tutorials (#8636)

I think Dr. Axel Rauschmeier is no longer issuing SSL certificates for his
website so https://speakingjs.com/es5/ gives a timeout error and on the
other side, http://speakingjs.com/es5/ is working fine.

digi-booster committed:
Replace broken link in Proxy doc (#8637)

https://reference.codeproject.com/book/dom/document/cookie/simple_document.cookie_framework
(the book directory is probably removed) ->
https://reference.codeproject.com/dom/document/cookie/simple_document.cookie_framework

cash2org committed:
Fix typo in onclick documentation (#8639)

sideshowbarker committed:
Add clarifications about the “for” attribute in <label> doc (#8642)

Fixes https://github.com/mdn/content/issues/8553

wbamberg committed:
[Markdown] [Web/API] Fix some unconvertible tables (#8588)

* Fix tables to be Markdownable

* Fix indentation

* Address review comments

sideshowbarker committed:
Make <input type="datetime-local"> example use real time zones (#8635)

Fixes https://github.com/mdn/content/issues/8631

digi-booster committed:
Replacing Broken Url in x-ms-acceleratorkey (#8641)

https://help.manuscript.com/7558/fogbugz-keyboard-shortcuts#For_Your_Server_or_non-Ocelot_Keyboard_Shortcuts -> https://support.fogbugz.com/hc/en-us/articles/360011254414-FogBugz-Keyboard-Shortcuts#:~:text=For%20Your%20Server%20or%20non-Ocelot%20Keyboard%20Shortcuts

Commit:
Fixes #5881 - adds a line about server push (#8643)

* Fixes #5881

* Update index.html

* Updates as suggested

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Commit:
Clarify in “Network Monitor” docs: it shows only HTTP requests (#8649)

janvidar committed:
Update text on regular expression flags (#8648)

digi-booster committed:
Fixing Url in OVR_multiview2 doc (#8651)

digi-booster committed:
Fixing Typo in FileSystemEntrySync doc (#8650)

teoli2003 committed:
Remove link to hit regions (#8655)

Follow-up for mdn/content#3780

mfuji09 committed:
fix RunTimeError with RuntimeError (#8656)

Commit:
Fixes #3674 - Corrects link txt for displaySurface (#8666)

Commit:
Fixes #5408 - Cannot modify JS from Tools (#8658)

hamishwillee committed:
agit status

hamishwillee committed:
Page templates  - API event/event handler to markdown

hamishwillee committed:
Glossary - bunch of prep for markdown conversion (#8590)

* Glossary - markdown prep: Remove styles
* Fix up notes/warnings
* Fix a bunch of seosummary class
* Apply suggestions from code review
* Try make github happy by adding a newline
Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Commit:
Fixes #7455 http to https inside images (#8559)

* Fixes 8143 shortens path to Demos

* Fixes 7455 updates images inline

* adds png images again and switches order of two links in SEE Also

* compresses files

hamishwillee committed:
Add brushes to pre blocks in Learn area (#8671)

digi-booster committed:
Fixing Url in <button>: The Button element ref (#8677)

https://a11yproject.com/posts/large-touch-targets/ -> https://www.a11yproject.com/posts/2018-11-21-large-touch-targets/

hamishwillee committed:
FF93 Release - AVIF updates (#8673)

rachelandrew committed:
adding small-caps to font-synthesis (#8680)

* adding small-caps to font-synthesis

* release notes

mathewhodson committed:
Use https for a demo link (#8698)

ChrisCrossCrash committed:
Use `onchange` instead of `addEventListener` (#8694)

The article is about `MediaQueryList.onchange`, but the example used `MediaQueryList.addEventListener`. I've corrected the example to use `onchange`.

mfuji09 committed:
remove unnecessary `<strong>` in Int16Array (#8702)

Elchi3 committed:
Document WebXR Hand Input part 1 (#8692)

Commit:
Fixes #8670 - Makes explicit the example reference (#8699)

* Fixes #8670 - Makes explicit the example reference

* Update files/en-us/web/javascript/reference/statements/async_function/index.md

Co-authored-by: wbamberg <will@bootbonnet.ca>

Co-authored-by: wbamberg <will@bootbonnet.ca>

Elchi3 committed:
WebXR transient input hit test: fix typo space -> profile (#8690)

mfuji09 committed:
Marking literals as inline codes (#8704)

dependabot[bot] committed:
chore(deps): bump @mdn/yari from 0.4.756 to 0.4.759 (#8689)

mfuji09 committed:
Reconstruct Global_Objects/Int32Array (#8703)

The content is for the Int32Array() constructor rather than Int32Array class.

queengooborg committed:
Remove PaymentDetailsUpdate dictionary (#8678)

* Remove PaymentDetailsUpdate dictionary

* Remove deprecated references

forevermatt committed:
Add return statements to "Clearing the cache" example code (#8701)

`waitUntil()` and `Promise.all()` each expect to be given a (or an array
of) Promise(s), but the missing `return`s meant that wasn't happening. This
page has the same code but with the returns:
https://developer.mozilla.org/en-US/docs/Web/API/CacheStorage/keys

Fixes #8695

Commit:
“Firefox user agent string reference”: Fix iPad/iPod/iPhone info (#8660)

Fixes #4779

wbamberg committed:
[Markdown] [Web/API] Make some more tables Markdownable (#8705)

queengooborg committed:
Remove onmozinterruptbegin/end from HTMLMediaElement page (#8706)

rachelandrew committed:
Release notes for ElementInternals.shadowRoot (#8686)

dependabot[bot] committed:
chore(deps): bump technote-space/get-diff-action from 4.2 to 5.0.1 (#8687)

Bumps [technote-space/get-diff-action](https://github.com/technote-space/get-diff-action) from 4.2 to 5.0.1.
- [Release notes](https://github.com/technote-space/get-diff-action/releases)
- [Changelog](https://github.com/technote-space/get-diff-action/blob/main/.releasegarc)
- [Commits](https://github.com/technote-space/get-diff-action/compare/v4.2...v5.0.1)

---
updated-dependencies:
- dependency-name: technote-space/get-diff-action
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump snok/install-poetry from 1.1.8 to 1.2 (#8688)

Bumps [snok/install-poetry](https://github.com/snok/install-poetry) from 1.1.8 to 1.2.
- [Release notes](https://github.com/snok/install-poetry/releases)
- [Commits](https://github.com/snok/install-poetry/compare/v1.1.8...v1.2)

---
updated-dependencies:
- dependency-name: snok/install-poetry
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Elchi3 committed:
Clarify requestSession & permissions; hand-tracking requires user consent (#8708)

teoli2003 committed:
Remove dictionary mention in StaticRange() (#8709)

teoli2003 committed:
Fixes links to Point -> WebKitPoint (#8710)

hamishwillee committed:
Basic docs for onsecuritypolicyviolation global handler et al (#8707)

hamishwillee committed:
API tag added to securitypolicyviolation_event (#8711)

teoli2003 committed:
Remove mentions of ReadOnly in (Animation)EffectTiming (#8712)

teoli2003 committed:
Remove mentions of outdated mixins in worker global scopes (#8717)

teoli2003 committed:
Remove non-standard non-implemented method in BluetoothDevice (#8713)

teoli2003 committed:
Fix redirects in HMDVRDevice.* (#8714)

teoli2003 committed:
Fix typos and redirected links in WebRTC (#8715)

teoli2003 committed:
Remove mention of XUL Node.baseURIObject (#8716)

teoli2003 committed:
Remove link to CSSMatrix (#8718)

dependabot[bot] committed:
chore(deps): bump @mdn/yari from 0.4.759 to 0.4.761 (#8725)

aniruddhadeshmukh2201 committed:
Grammatical correction in index.html (#8727)

changing Line no. 13 of index.html
"This aim of this skill test is to assess whether you've ..."
to 
"The aim of this skill test is to assess whether you've..."

bartek-szymanski committed:
Update index.md (#8732)

Missing letter `s` in the word `shows`.

escattone committed:
remove peterbe from CODEOWNERS (#8733)

Commit:
Fixes #5614 removes bytestring (#8740)

Commit:
Fixes #8723 - Removes details of external UI's (#8735)

* Fixes #8723 - Removes details of external UI's

* Delete font2.png

* Delete font1.png

* Clarifies which details

queengooborg committed:
Remove RTCIceCandidateInit dictionary from MDN content (#8721)

Commit:
Fix constructors with multiple sets of arguments. (#8572)

* Fix constructors with multiple sets of arguments.

* Fix http.

* Update files/en-us/web/api/passwordcredential/passwordcredential/index.html

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/web/api/keyframeeffect/keyframeeffect/index.html

Co-authored-by: Rachel Andrew <me@rachelandrew.co.uk>

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>
Co-authored-by: Rachel Andrew <me@rachelandrew.co.uk>

fowl2 committed:
HTMLCollection: warn about modifying while iterating (#8742)

Elchi3 committed:
Document XRHand (#8724)

Rumyra committed:
Updating CODEOWNERS (#8728)

* rel notes

* updating codeowners

Co-authored-by: Ruth John <ruthjohn@Ruths-Work-Mac.local>

Rumyra committed:
Editing FF92 release notes (#8685)

* rel notes

* ff92 rel notes

Co-authored-by: Ruth John <ruthjohn@Ruths-Work-Mac.local>

queengooborg committed:
Merge KeyboardEvent/MouseEvent.which into UIEvent.which (#8751)

Commit:
Fixes #8743 - Removes json.parse polyfill (#8745)

dependabot[bot] committed:
chore(deps): bump @mdn/yari from 0.4.761 to 0.4.763 (#8753)

mfuji09 committed:
Fix links in BigInt.toLocaleString() (#8754)

digi-booster committed:
fixing typo in letter-spacing css ref (#8752)

digi-booster committed:
Adding note and Experimental header note line-height-step css ref (#8747)

* Adding note and Experimental header note line-height-step css ref

* Update index.md

* Update index.md

wbamberg committed:
[Markdown] [Web/API] Suppress GFM conversion of wide (>150 character) tables (#8736)

Commit:
Fixes #8761 - Extra "is supported" near EOL (#8763)

Commit:
Fix minor grammatical error (#8757)

Anurella committed:
Example using Array.prototype.fill() on empty array (#7429)

* Added an example using the :scope pseudo class

* removed changes.md

* removed changes.md

* removed package-lock file

* adjusted to fit review suggestion

* remove file only to be used on local

* Added a note and example for using fill() with empty array

* Edited to adjust suggestions

* Merged mdn/main

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

trrapp12 committed:
Adds some information about setTimeout() (#8764)

LosManos committed:
Clarify 'unsafe-hashes' (#6490)

psea committed:
Refine HTTP request examples in “Client-Server Overview” (#8726)

danieldafoe committed:
docs: use simpler language for row- and column-reverse descriptions (#8762)

The word "permuted" was being used. This tripped me up so much while reading that I had to stop and open this PR.

This is not very approachable language. It's not a commonly used word, so let's try to improve the reading level for these descriptions. 

Open to suggestions or improvements for my proposed change! 🙂

Commit:
Fixes #2645: adds overview entry to groupdata.json for Screen Orienta… (#8765)

sideshowbarker committed:
selection.selectAllChildren() isn’t experimental (#8767)

Fixes https://github.com/mdn/content/issues/8766

Related BCD PR: https://github.com/mdn/browser-compat-data/pull/12311

Elchi3 committed:
Document XRJointPose & XRJointSpace (#8770)

lukewarlow committed:
Add interactive example for accent-color (#8730)

dependabot[bot] committed:
chore(deps): bump @mdn/yari from 0.4.763 to 0.4.765 (#8772)

rachelandrew committed:
Adds CustomStateSet (#8093)

* Adds CustomStateSet

* Adding set methods (#2)

* Adding set methods

* case

* and back

* Apply suggestions from code review

Co-authored-by: Joe Medley <jmedley@google.com>

Co-authored-by: Joe Medley <jmedley@google.com>

Purecoder committed:
Update index.html (#8781)

thestelz committed:
fix: window typo (#8778)

* fix: window typo

Fix `Window` to `window` in the syntax example for copy, paste.

* feat: remove syntax section

Remove Syntax section per feedback.

updates mdn/content#8778

Anurella committed:
Showing the CSS for the synth keyboard example (#7516)

* Showing the CSS for the synth keyboard example

* Suggested review adjustment

Co-authored-by: wbamberg <will@bootbonnet.ca>

codegirl-007 committed:
Add more examples of using URLSearchParams (#7218)

* Add more example of using URLSearchParams

This commit adds:
- an example of using an object literal as a parameter.
- an explanation when you have duplicate parameters in a query string

* Remove leftover quotation mark

* Change to single-quote here

* Update files/en-us/web/api/urlsearchparams/index.html

Co-authored-by: wbamberg <will@bootbonnet.ca>

* Update files/en-us/web/api/urlsearchparams/index.html

Co-authored-by: wbamberg <will@bootbonnet.ca>

* Split example into three parts; align comments; fix one expected output

Co-authored-by: wbamberg <will@bootbonnet.ca>

queengooborg committed:
Remove DOMPointInit from MDN docs (#8788)

* Remove DOMPointInit.w/x/y/z and redirect

* Fold DOMPointInit's information into DOMPoint[ReadOnly].fromPoint

* Delete DOMPointInit page

* Remove references to DOMPointInit

wbamberg committed:
Markdown doc table updates (#8787)

* Update Markdown guide to talk about table max width and properties tables

* some cleanup

wbamberg committed:
Fix a dl closing tag (#8776)

mathewhodson committed:
Make an anchor lowercase on letter-spacing page (#8779)

This fixes a broken link flaw reported by yari.

huyenltnguyen committed:
docs: update code formatting (#8789)

teoli2003 committed:
Remove 2 non-documented mixins in InterfaceData.json (#8287)

Konrud committed:
Move relevant information to the top of the page (#8792)

_Difference between :is() and :where()_ Should appear at the top of the page.

Rumyra committed:
Adding hacks post to FF92 release notes (#8774)

* rel notes

* adding hacks to rel notes

Co-authored-by: Ruth John <ruthjohn@Ruths-Work-Mac.local>

lex111 committed:
Fix incorrect Markdown (#8795)

Elchi3 committed:
Replace some WebXR tables with lists (#8799)

bal360 committed:
Updated uni-form resource link (#8791)

blue-jam committed:
Fix typo in background and borders (#8805)

CSharperMantle committed:
Fix incorrectly anchored link to RFC 7231 (#8809)

digi-booster committed:
Enter key in <length> css ref (#8800)

* Enter key in <length> css ref

I just want to say that the Enter key is a more popular name than the Return key except in Norway.

* implement suggestions.

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Myndex committed:
Add link to color article (#8777)

annevk committed:
Purported Firefox CORP + PDF bug is marked WFM (#8811)

Commit:
Fixes #961 - Mod and remainder are not the same (#8814)

dependabot[bot] committed:
chore(deps): bump @mdn/yari from 0.4.765 to 0.4.770 (#8816)

h0t-p1xels committed:
Add closing `</p>` tag in example (#8817)

Replace opening `<p>` tag with closing `</p>` tag in example

lucacasonato committed:
Add Crypto.randomUUID page (#8045)

* Add Crypto.randomUUID page

This adds a page for the new Crypto.randomUUID API, proposed in this
spec: https://wicg.github.io/uuid.

It is shipped in Chrome 92, Deno 1.11 and Node 16.7.0.

* typo

* fixes

* add specification and compat sections

* fix

Commit:
Fixes #941 - Matches endIndex, includes NaN (#8820)

teoli2003 committed:
Remove miniDaemon documentation from setInterval (#8810)

Fixes #8798

AnilSeervi committed:
add missing <head> tag and rearrange closing tag (#8667)

* add missing <head> tag and rearrange closing tag

* add quotes

* Initial attempt converting to live sample

* Changes for live sample

* check onload event not registering

* try events

* change live sample height

* revert variable name

* review changes

* add reload button

* Update files/en-us/web/api/globaleventhandlers/ondragenter/index.html

* Update files/en-us/web/api/globaleventhandlers/ondragenter/index.html

Co-authored-by: wbamberg <will@bootbonnet.ca>

teoli2003 committed:
Remove <code> in <pre> +  link to same page (#8819)

Elchi3 committed:
WebXR: convert a wide table back to HTML (#8801)

Commit:
Fix heading after syntax removed. (#8818)

outspokendev committed:
Add missing backtick to line 50 (#8825)

The missing backtick causes the remaining portions of the page to render as a codeblock.

Anindya-Prithvi committed:
Inconsistent "In this module" section on debugging page (#8822)

The page https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Debugging_HTML
should conform the style on page https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Marking_up_a_letter

teoli2003 committed:
Create Layout Instability Sidebar (#8803)

rachelandrew committed:
Release notes and a small update to the ResizeObserver page (#8755)

* Release notes and a small update to the ResizeObserver page

* Close `<ul>`

Co-authored-by: wbamberg <will@bootbonnet.ca>

digi-booster committed:
Remove react360 url in Getting started with React guide (#8719)

Facebook has now archived and stopped supporting React 360 projects but is still available to view and use.

Elchi3 committed:
Remove WindowOrWorkerGlobalScope mixin page (#8796)

* Remove WindowOrWorkerGlobalScope mixin page

* Remove links

mfuji09 committed:
Update CSS/calc() (#8851)

wbamberg committed:
Fix strong inside em (#8849)

mfuji09 committed:
Fix a link (#8843)

CSharperMantle committed:
Add technical reference for opengl.org (#8840)

autumnontape committed:
Update XML's entry in list of common MIME types (#8848)

The distinction between `text/xml` and `application/xml` is obsolete as of RFC 7303, and `application/xml` is recommended. Also add a note about the possibility that more specific MIME types may apply to XML files and remove the unnecessary backticks around "XML."

TyMick committed:
Add IDB to list of IndexedDB libraries (#8844)

I placed it third in the list, as the list was already arranged in descending order by number of GitHub stars.

Ngoakor12 committed:
updated description of anonymous function (#8841)

Hi, I updated the description to properly define the examples. "Traditional function" to me usually has a name, so when the documentation used "traditional function" I was a bit confused as it really wasn't and I thought it might also be confusing to most beginners who haven't encountered anonymous functions.

mfuji09 committed:
minor fixes in Statements/async_function (#8845)

Commit:
Fixes #5705 - setInterval's func's first execution (#8855)

AmirDoreh committed:
Fix ICE acronym description (#8853)

* Update index.html

ICE => Interactive Connectivity Establishment

* Update files/en-us/web/api/rtcicecandidate/index.html

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

deontic committed:
Use keydown instead of the deprecated keypress event (#8862)

Make the example code at https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API work with `keydown` instead of the deprecated `keypress` .
This will prevent beginners or people otherwise learning from or using the example code from using a deprecated event, which is generally bad practice.

digi-booster committed:
add "of" in @import css ref (#8868)

Commit:
Fixes #3955 - pretty prints forbidden samples (#8859)

* Fixes #3955 - pretty prints forbidden samples

* Adds example-bad to js

Commit:
Fixes #4120: details about context of another doc (#8860)

* Fixes #4120: details about context of another doc

* Removes h4 from Note

deontic committed:
Update index.md (#8863)

Fixes #8861 by using the correct parameter instead of `index`.

Elchi3 committed:
Document XRFrame.fill{JointRadii,Poses} (#8815)

teoli2003 committed:
Precise context of toString() (#8837)

Fixes #8830

jamesaucode committed:
Fix Re-exporting / Aggregating section of export (#8852)

jamesaucode committed:
Add static getters example (#8850)

* add static getter constants example

* small modification for the title

teoli2003 committed:
Remove remaining mixins from InterfaceData (#8854)

anton-bot committed:
typo: isNan => isNaN (#8870)

hamishwillee committed:
FF93 KeyboardEvent.initKeyEvent() hidden behind preference (#8793)

* FF93 KeyboardEvent.initKeyEvent() hidden behind preference

* Add release note

queengooborg committed:
Remove AudioNodeOptions dictionary from MDN (#8867)

* Remove AudioNodeOptions dictionary from MDN

* Fix indentation

* Fix redirect

Commit:
Fixes #4285 - Rewords desc for omitting args (#8856)

* Fixes #4285 - Rewords desc for omitting args

* Update files/en-us/web/api/canvasrenderingcontext2d/drawimage/index.html

Co-authored-by: Florian Scholz <fs@florianscholz.com>

Co-authored-by: Florian Scholz <fs@florianscholz.com>

vandijkstef committed:
add reportValidity (#8821)

* add reportValidity

I was missing information on how to trigger the native browser validation messages while dealing with a form onSubmit handler. Additionally, the methods are available to form elements as well beside the form fields

* Fix grammar.

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

rachelandrew committed:
add attachInternals to release notes for 93 (#8872)

* add attachInternals to release notes for 93

* Update files/en-us/mozilla/firefox/releases/93/index.html

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>
Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

hamishwillee committed:
FF93 Release notes: global onsecuritypolicyviolation and onslotchange handlers (#8785)

* FF93 RelNotes: add global onsecuritypolicyviolation handler

* FF93 Relnote for global onslotchange handler

* Fix up broken macro call

Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

wbamberg committed:
Replace alert() in Web/API live samples (#7570)

* Example replacement for alert

* Address review comments

rachelandrew committed:
fixing broken examples in CSS Paint API guide (#8395)

* fixing broken examples

* Update files/en-us/web/api/css_painting_api/guide/index.html

Co-authored-by: wbamberg <will@bootbonnet.ca>

Co-authored-by: wbamberg <will@bootbonnet.ca>

hamishwillee committed:
securitypolicyviolation events only work on Window/Document (#8784)

Commit:
Fixes #4124 - adds console logs (#8858)

clavicus committed:
Array map syntax fixes (#8873)

* Removes unnecessary whitespace

* Add thisArg to arrow function syntax

danr committed:
Fix then->than in grid-template-columns (#8874)

rachelandrew committed:
updating datetime-local (#8876)

* updating datetime-local

* compress image

johnjinto committed:
Closing div tag of hidden div block relocated. (#8568)

The closing div tag is moved to enclose the repeating css rule of "column-width: 200px;" in the hidden block. Currently this css code snippet is repeatedly appearing.

Biki-das committed:
fixed edge screenshot (#8571)

sarahannnicholson committed:
Adding missing theme colors (#8573)

This list of theme colors in High Contrast mode was great! However, just missing a few. I have added the remaining ones.

clavicus committed:
Array map syntax  (#8875)

* Removes unnecessary whitespace

* Add thisArg to arrow function syntax

* Removing thisArg from arrow function syntax

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

gibson042 committed:
compareDocumentPosition: Add bit descriptions and move the table into Return value (#8865)

* compareDocumentPosition: Add bit descriptions and move the table into Return value

* Fix typo

* Fix broken links

* Revert accidental change

Ref b26001dcf59ff9e327d14d802221c1fc209baa79

* Fix macro error breaking tests

Co-authored-by: Florian Scholz <fs@florianscholz.com>

* Fix bitwise_AND link

* Fix link

* 2nd attempt

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>
Co-authored-by: Florian Scholz <fs@florianscholz.com>

hamishwillee committed:
FF93: Add onslotchange global handler (#8790)

* Add onslotchange global handler

* Add ShadowRoot onslotchange

* Remove syntax block

* Remove syntax block

dependabot[bot] committed:
chore(deps): bump @mdn/yari from 0.4.770 to 0.4.771 (#8878)

TurekBot committed:
Add Spring as Java server framework (#8827)

It seems to make sense to include a framework for Java since it's the most used language in the world currently and is often taught in schools. 

Spring has a quick-start tutorial specifically about handling form submission which seems like a really good fit.

Commit:
Fixes #1630: removes incorrect page about 'use strict' (#8877)

* Fixes #1630: removes page

* Revert "Fixes #1630: removes page"

This reverts commit 82aa9596d9e6157a74a1252baf5e512c77538f56.

* removes var_hides_argument

queengooborg committed:
Remove Transferable from MDN (#8880)

* Remove Transferable from MDN

* Remove redirect

deontic committed:
Fix IDBCursorWithValue broken links (#8881)

Update broken links

mfuji09 committed:
Add missing Specifications section (#8882)

jasonflorentino committed:
Fix typo: Plural vs Singular (#8879)

Fix plural nouns that should be singular or vice versa.

rebloor committed:
Merge branch 'Bug-1612390-sidebars-included-in-extension.getViews-when-windowId-specified' of ssh://github.com/rebloor/content into Bug-1612390-sidebars-included-in-extension.getViews-when-windowId-specified

rebloor commented:
@Rob--W I'm not sure on getting anywhere useful with my attempts to rebase, so closing this one and will redo.
